typescript/introduction/tsc-data-types-ii/tsc-object-type-comparison.md

```diff
@@ -0,0 +1,124 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# `Object` vs `object` vs `{}`
+
+---
+## Content
+
+TypeScript has 3 ways to represent an object with slightly different behaviors.
+
+Since almost everything in JavaScript is an object[1], the `Object` type can be used to represent any type that is considered to contain a value:
```

Comment:
```suggestion
Since any defined value in JavaScript is an object[1], the `Object` type can be used to represent any type that is considered to contain a value:
```